### ISSUE TITLE
Downloaded 1.2.0 and now the selection tool is malfunctioning

### ISSUE THREAD

**Issue by setonlazalier** [tabulapdf/tabula#844]
I just downloaded the 12.0 version and now the selection tool is malfunctioning - it correctly highlights my selections at the time that I initially make them, but when I preview and export, it is cutting off the last column of information. When I go back to view my selections again, they have shifted to the left of the page, which explains the missing information. The document I am using is the same one that I use repeatedly throughout the year, with the same selection/style each time - one large box that basically spans the entire page of the PDF.

Certainly seems to be related directly to 12.0.. will try to download an older version and work from that for now. Thanks!

**Comment by jeremybmerrill** [comment]
Hi @setonlazalier

I don't see the issue of selections moving. Can you share the document you're using, in case that's related to the problem? (If it's private, you can email me, at my username at gmail.)

I suspect the problem with the missing last column is not related to the moving selections. Does the PDF have ruling lines surrounding each column and is the Lattice button highlighted on the left sidebar of the Preview and Extract screen? I'm suspect it is; in Lattice mode, all of the ruling lines must be included in the selection for it to work. Try expanding your selection to include the ruling lines and see if that gets the last column to be included.

**Comment by setonlazalier** [comment]
Hi Jeremy,

So sorry I did not get back to you earlier. 

The PDF selections I was having trouble with did include all ruling lines and tabula was correctly parsing and identifying the ruling lines using lattice mode, but it was a glitch with the actual placement of the selection box itself - it would shift several spaces to the left, cutting out whatever was to the right of it, no matter how I modified the size/location of the selection box. 

Good news is that I downloaded the latest update 1.2.1 and that totally solved the selection issue I was having with 1.2.0 so it looks like that was remedied in the update. If it would be helpful for you, I can go back using 1.2.0 and screen record to try and recreate the glitch I was running into with my pdfs.

Thanks for your response on this!

Seton
